### PR TITLE
MS17380: Create App fix for angularVelocity zeroing out on duplication or undo

### PR DESCRIPTION
**Pull request by thoys**
Fixes https://highfidelity.manuscript.com/f/cases/17380/QuickFix-localAngularVelocity-in-Edit-js

## QA Test

1. Open the Create App in desktop mode
2. Create a Box
3. Set the Angular Damping to 0
4. Set the Angular Velocity to 5 on any random axis
5. Move the Box using the edit handles
6. Press `Ctrl + Z` to undo the move.
7. The Box should still be spinning.
8. Duplicate the box using the alt key and draging the entity on one of the move handles.
9. Both Boxes should be spinning

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11938/HighFidelity-Beta-PR13781-9c96a10.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11938/HighFidelity-Beta-PR13781-9c96a10.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11938/HighFidelity-Beta-PR13781-9c96a10.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11938/HighFidelity-Beta-Interface-PR13781-9c96a10.exe

File: scripts/system/libraries/entitySelectionTool.js [hunk 1]
```diff
@@ -28,6 +28,17 @@ Script.include([
 SelectionManager = (function() {
     var that = {};
 
+    /**
+     * @description Removes known to be broken properties from a properties object
+     * @param properties
+     * @return properties
+     */
+    var fixRemoveBrokenProperties = function (properties) {
```
Comment (dback2) [inline scripts/system/libraries/entitySelectionTool.js]: PR is building for the updated changes, and test plan adjusted accordingly, will give Peter a heads up as well

Comment (dback2) [inline scripts/system/libraries/entitySelectionTool.js]: Would this be beneficial as a library/global function instead or is it only useful here? Or it's only a temp workaround for angular velocity that we intend to be fixed soon

**Comment by QliCknox** [comment]
Currently Testing

**Comment by qliacoombs** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12093/HighFidelity-Beta-PR13781-9c96a10.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12093/HighFidelity-Beta-PR13781-9c96a10.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12093/HighFidelity-Beta-Interface-PR13781-9c96a10.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12093/HighFidelity-Beta-PR13781-9c96a10.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12093/HighFidelity-Beta-Interface-PR13781-9c96a10.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Pass @conklin94122 

Velocity stays constant after undoing or duplicating the entity

**Pull request merged**